bridge/scripts/tasks/alicenetTasks.ts

```diff
@@ -1750,6 +1751,251 @@ task("update-alicenet-node-version", "Set the Canonical AliceNet Node Version")
+    // encode ALCA amount in wei
+    const alcaAmount = hre.ethers.utils.parseEther(taskArgs.alcaAmount);
+    let promptMessage = `approve publicStaking ${
+      publicStaking.address
+    } to spend  ${alcaAmount.toString()} in wei ALCA ? ${hre.ethers.utils.parseEther(
+      alcaAmount.toString()
+    )}(y/n)\n`;
+    await promptCheckDeploymentArgs(promptMessage);
+    const approval = alca.interface.encodeFunctionData("approve", [
+      publicStaking.address,
+      alcaAmount,
+    ]);
+    const encodedApprovalCall: MultiCallArgsStruct = {
+      target: alca.address,
+      value: 0,
+      data: approval,
+    };
+    // encode mintTo call to public staking contract to mint a staked position with ALCA from factory
+    const mintTo = publicStaking.interface.encodeFunctionData("mintTo", [
+      taskArgs.recipient,
+      alcaAmount,
+      maxStakingLock,
+    ]);
+    promptMessage = `create a staked position for ${
+      taskArgs.recipient
+    } with ${alcaAmount} in wei ${hre.ethers.utils.parseEther(
+      alcaAmount.toString()
+    )} ALCA locked for ${maxStakingLock} blocks? (y/n)\n`;
+    await promptCheckDeploymentArgs(promptMessage);
+    const encodedMintToCall: MultiCallArgsStruct = {
+      target: publicStaking.address,
+      value: 0,
+      data: mintTo,
+    };
+    const encodedMultiCallArgs: Array<MultiCallArgsStruct> = [
+      encodedApprovalCall,
+      encodedMintToCall,
+    ];
+    const balanceBefore = await hre.ethers.provider.getBalance(
+      factory.signer.getAddress()
+    );
+    const gas = await factory.estimateGas.multiCall(
+      encodedMultiCallArgs,
+      await getGasPrices(hre.ethers)
+    );
+    promptMessage = `total gas required is ${gas.toString()} Continue? (y/n)\n`;
+    await promptCheckDeploymentArgs(promptMessage);
+    const txResponse = await factory.multiCall(
+      encodedMultiCallArgs,
+      await getGasPrices(hre.ethers)
+    );
+    const receipt = await txResponse.wait(waitConfirmationsBlocks);
+    const balanceAfter = await hre.ethers.provider.getBalance(
+      factory.signer.getAddress()
+    );
+    if (taskArgs.test) {
+      console.log(`total cost = ${balanceBefore.sub(balanceAfter)}`);
+    }
+    console.log(receipt.events);
+  });
+
+task(
+  "monthly-distribution",
+  "takes in a csv file with addresses, and ALCA amount"
+)
+  .addParam("csvPath", "path to csv file")
+  .addFlag("test", "test mode for use with a hardhat fork mode")
+  .addOptionalParam(
+    "waitConfirmation",
+    "wait specified number of blocks between transactions",
+    0,
+    types.int
+  )
+  .setAction(async (taskArgs, hre) => {
+    const waitConfirmationsBlocks = await parseWaitConfirmationInterval(
+      taskArgs.waitConfirmation,
+      hre
+    );
+    const maxStakingLock = BigNumber.from("1051200");
+    let factory = await hre.ethers.getContractAt(
+      ALICENET_FACTORY,
+      "0x4b6dF6B299fB6414f45719E0d9e1889269a7843E"
```

Comment:
Same as above, maybe using the factory address as a parameter and deriving the other address below would be more robust. But I'm fine with hardcoding this if the only intent is to use it on mainnet. Maybe use these addresses as constants in the constant file?